<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>798</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>798</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="run">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string> [ Run ]</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>761</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QFrame" name="neurons_layer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>621</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="n_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>240</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="n_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="n_1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="n_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>200</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Neurons - Layer 3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>200</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Neurons - Layer 4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Neurons - Layer 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Neurons - Layer 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>310</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Neurons - Layer 5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="n_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>350</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QFrame" name="img_layer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>621</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="img_x">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="img_y">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="img_dim">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>180</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>X_Size</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y_Size</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dim_Size</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="function_layer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>621</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="a_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>240</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="a_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="a_1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="a_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Activation - Layer 1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="a_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>350</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Activation - Layer 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Activation - Layer 3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>200</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Activation - Layer 4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>310</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Activation - Layer 5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="padding">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="train_frame">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>19</y>
      <width>120</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="model_frame">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>111</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="image_size">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Images Size</string>
     </property>
    </widget>
    <widget class="QPushButton" name="neurons">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Neurons</string>
     </property>
    </widget>
    <widget class="QPushButton" name="functions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Functions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="compile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Compile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="main_frame">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>120</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="model">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
    <widget class="QPushButton" name="train">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Train</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="compile_layer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>621</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="loss">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="optimizer">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="metrics">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>180</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Loss</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Optimizer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>130</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Metrics</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="train_main_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>621</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="epochs">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>330</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="validation_split">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="b_size">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="verbose">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_train">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>230</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Validation Split</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Batch Size</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>290</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Epochs</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Verbose</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>190</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Name *</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>290</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Filepath</string>
     </property>
    </widget>
    <widget class="QPushButton" name="filepath_train">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>330</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="back_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <zorder>train_main_frame</zorder>
   <zorder>function_layer</zorder>
   <zorder>neurons_layer</zorder>
   <zorder>img_layer</zorder>
   <zorder>compile_layer</zorder>
   <zorder>model_frame</zorder>
   <zorder>line</zorder>
   <zorder>progressBar</zorder>
   <zorder>run</zorder>
   <zorder>label</zorder>
   <zorder>train_frame</zorder>
   <zorder>main_frame</zorder>
   <zorder>back_</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
